test/cmdlineTests.sh

```diff
@@ -310,6 +310,9 @@ function test_solc_assembly_output
 
 function test_via_ir_equivalence()
 {
+    SOLTMPDIR=$(mktemp -d)
+    cd "$SOLTMPDIR"
```

Comment:
```suggestion
    pushd "$SOLTMPDIR"
```
And then `popd` at the end of the function. So that calling the function still preserves your working dir.

Reply:
Good point! Code updated accordingly.